db/migrations/20201028150700_podcasts.js

```diff
@@ -3,17 +3,18 @@ exports.up = function (knex) {
     return knex.schema
       .createTable('podcasts', function (table) {
         table.increments('id').primary();
-        table.string('podcast_id');
         table.string('podcast_name');
         table.string('podcast_title');
         table.string('author');
-        table.string('audience');
+        table.boolean('familiy_friendly').defaultTo(1);
         table.string('genre');
-        table.string('collectionViewUrl');
+        table.string('type');
+        table.string('collection_url');
         table.integer('episode_count');
         table.string('release_date');
         table.string('image_60');
         table.string('image_100');
+        table.timestamps(true, true)
```

Comment:
I like the idea of keeping timestamps consistent across for all our tables, to assure the batching is consistent across the board, and if we ever dome sort of randomization we can see when it was last done .

Reply:
I agree, this will provides us with important information moving forward.

Reply:
100% agree!